tasks/concurrent.js

```diff
@@ -14,6 +15,8 @@ module.exports = function (grunt) {
 			limit: Math.max((os.cpus().length || 1) * 2, 2)
 		});
 		var tasks = this.data.tasks || this.data;
+		var maxTaskLength = Math.max.apply(null,(tasks.map(function(task){ return task.length})));
+		var colors = ["blue","magenta","yellow","green","red","cyan","white","gray"];
```

Comment:
We can't use `white` either as that's the default color in most terminals and will conflict with the non-styled tasks.